### ISSUE TITLE
TestCaseSourceAttribute constructor for method with parameters

### ISSUE THREAD

**Issue by oznetmaster** [nunit/nunit#1765]
TestCaseSourceAttribute has a constructor that takes a string with the name of the method/field/property that provides it data, and a constructor that takes type of the source, a string with its name, and parameters if it is a method,

It would be convenient if a constructor that took just the string name of the method and parameters were available as well,

``TestCaseSourceAttribute (string sourceName, object[] methodParams)``

**Comment by CharliePoole** [comment]
Makes sense to me. I made it low priority, but if it's higher for you, feel free to do a PR.
